Dockerfile-centos

```diff
@@ -0,0 +1,36 @@
+FROM centos:7
+MAINTAINER Bryan Latten <latten@adobe.com>
+
+# Use in multi-phase builds, when an init process requests for the container to gracefully exit, so that it may be committed
+# Used with alternative CMD (worker.sh), leverages supervisor to maintain long-running processes
+ENV SIGNAL_BUILD_STOP=99 \
+    S6_BEHAVIOUR_IF_STAGE2_FAILS=2 \
+    S6_KILL_FINISH_MAXTIME=5000 \
+    S6_KILL_GRACETIME=3000 \
+    S6_VERSION=v1.18.1.5 \
+    GOSS_VERSION=v0.2.5
+
+# Ensure scripts are available for use in next command
+# COPY ./container/root/security_updates.sh / \
```

Comment:
If you need this, just do `yum upgrade -y` for `security_updates.sh` then for clean you can do: `yum clean all; yum clean metadata` to keep layers small.